spec/models/task_spec.rb

```diff
@@ -29,6 +30,11 @@
       expect(task).not_to be_valid
     end
 
+    it "is not valid without existing project" do
+      task = FactoryGirl.build(:task)
+      task.save!(validate: false)
```

Comment:
It works that way ;) when I change it to `(validate: true)`, it won't pass the test